Source-Code/Drone2/.spreadsheet_view.ini

```diff
@@ -0,0 +1,76 @@
+[General]

```

Comment:
Do we need to upload these files?  If not we can just have them on the .gitignore (if they aren't already) so we don't have extra unnecessary files in the repo.